src/Serializer/Handler/DependsHandler.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace DavidBadura\Taskwarrior\Serializer\Handler;
+
+use DavidBadura\Taskwarrior\Exception\ReferenceException;
+use DavidBadura\Taskwarrior\Task;
+use DavidBadura\Taskwarrior\TaskManager;
+use Doctrine\Common\Collections\ArrayCollection;
+use JMS\Serializer\Context;
+use JMS\Serializer\GraphNavigator;
+use JMS\Serializer\Handler\SubscribingHandlerInterface;
+use JMS\Serializer\VisitorInterface;
+
+/**
+ * @author David Badura <d.a.badura@gmail.com>
+ */
+class DependsHandler implements SubscribingHandlerInterface
+{
+    /**
+     * @var TaskManager
+     */
+    protected $taskManager;
+
+    /**
+     * @param TaskManager $taskManager
+     */
+    function __construct(TaskManager $taskManager)
+    {
+        $this->taskManager = $taskManager;
+    }
+
+    /**
+     * @return array
+     */
+    public static function getSubscribingMethods()
+    {
+        $methods = array();
+
+        $methods[] = array(
+            'type'      => 'Depends',
+            'format'    => 'json',
+            'direction' => GraphNavigator::DIRECTION_DESERIALIZATION,
+            'method'    => 'deserializeCarbon'
+        );
+
+        $methods[] = array(
+            'type'      => 'Depends',
+            'format'    => 'json',
+            'direction' => GraphNavigator::DIRECTION_SERIALIZATION,
+            'method'    => 'serializeCarbon'
+        );
+
+        return $methods;
+    }
+
+    /**
+     * @param VisitorInterface $visitor
+     * @param Task[] $tasks
+     * @param array $type
+     * @param Context $context
+     * @return string
+     * @throws ReferenceException
+     */
+    public function serializeCarbon(VisitorInterface $visitor, $tasks, array $type, Context $context)
```

Comment:
Correct me if I am wrong, but this commit handles task dependencies, right? `task 12 depends:42`

These method names seem like a copy/paste mistake from special serialization/deserialization from the date carbon stuff, or am I missing something?